javav2/example_code/kinesis/src/main/java/com/example/kinesis/KinesisStreamEx.java

```diff
@@ -58,7 +48,7 @@ public static void main(String[] args) {
 
         SubscribeToShardRequest request = SubscribeToShardRequest.builder()
                 .consumerARN(CONSUMER_ARN)
-                .shardId("arn:aws:kinesis:us-east-1:814548047983:stream/StockTradeStream")
+                .shardId("arn:aws:kinesis:us-east-1:000008047983:stream/StockTradeStream")

```

Comment:
This looks like a real account ID. Use approved fictitious numbers from the style guide, like 111122223333.

Reply:
fixed